packages/datagateway-search/src/card/datasetSearchCardView.component.tsx

```diff
@@ -0,0 +1,311 @@
+            )
+          : tableLink(
+              `/browse/instrument/${instrumentId}/facilityCycle/${facilityCycleId}/investigation/${datasetData.investigation.id}/dataset/${datasetData.id}`,
+              datasetData.name
+            );
+      }
+      return linkType === 'investigation' ? '' : datasetData.name;
+    },
+    [facilityCycles]
+  );
+
+  const genericLink = (
+    datasetData: Dataset,
+    linkType = 'dataset'
+  ): React.ReactElement | string => {
+    if (datasetData.investigation) {
+      return linkType === 'investigation'
+        ? tableLink(
+            `/browse/investigation/${datasetData.investigation.id}/dataset`,
+            datasetData.investigation.name
+          )
+        : tableLink(
+            `/browse/investigation/${datasetData.investigation.id}/dataset/${datasetData.id}/datafile`,
+            datasetData.name
+          );
+    }
+    return linkType === 'investigation' ? '' : datasetData.name;
+  };
+
+  const hierarchyLink = React.useMemo(() => {
+    if (hierarchy === 'dls') {
+      return dlsLink;
+    } else if (hierarchy === 'isis') {
+      return isisLink;
+    } else {
+      return genericLink;
+    }
+  }, [hierarchy, isisLink]);
+
+  // hierarchy === 'isis' ? data : [] is a 'hack' to only perform
+  // the correct calculation queries for each facility
+  const datasetCountQueries = useDatasetsDatafileCount(
+    hierarchy !== 'isis' ? data : []
+  );
+  const sizeQueries = useDatasetSizes(hierarchy === 'isis' ? data : []);
+
+  const title = React.useMemo(
+    () => ({
+      // Provide label for filter component.
+      label: t('datasets.name'),
+      // Provide both the dataKey (for tooltip) and content to render.
+      dataKey: 'name',
+      content: (dataset: Dataset) => {
+        return hierarchyLink(dataset);
+      },
+      filterComponent: textFilter,
+    }),
+    [hierarchyLink, t, textFilter]
+  );
+
+  const description = React.useMemo(
+    () => ({
+      label: t('datasets.details.description'),
+      dataKey: 'description',
+      filterComponent: textFilter,
+    }),
+    [t, textFilter]
+  );
+
+  const information = React.useMemo(
+    () => [
+      {
+        icon: ConfirmationNumber,
+        label:
+          hierarchy === 'isis'
+            ? t('datasets.size')
+            : t('datasets.datafile_count'),
+        dataKey: hierarchy === 'isis' ? 'size' : 'datafileCount',
+        content: (dataset: Dataset): string => {
+          const index = data?.findIndex((item) => item.id === dataset.id);
+          if (typeof index === 'undefined') return 'Unknown';
+          const query =
+            hierarchy === 'isis'
+              ? sizeQueries[index]
+              : datasetCountQueries[index];
+          return formatCountOrSize(query, hierarchy === 'isis');
+        },
+        disableSort: true,
+      },
+      {
+        icon: Fingerprint,
+        label: t('datasets.investigation'),
+        dataKey: 'investigation.name',
```

Comment:
This should match the table and I think this needs to be `title` right (due to my previous comment)?